Code/GraphMol/FileParsers/file_parsers_catch.cpp

```diff
@@ -3019,6 +3020,47 @@ TEST_CASE("test bond flavors when writing PDBs", "[bug]") {
   }
 }
 
+TEST_CASE("test output with incomplete monomer info", "[bug][writer]") {
```

Comment:
Since you've put the `setw()` in for the residue name, I think it would make sense to add a test for that here too: something that sets the residue name to a string with more than three characters and the verifies that the output PDB only includes the first 3 of those.